src/search/command_line.cc

```diff
@@ -47,28 +47,26 @@ static vector<string> replace_deprecated_predefinitions(const vector<string> &ar
         bool is_last = (i == args.size() - 1);
 
         if (arg == "--evaluator" || arg == "--heuristic" || arg == "--landmarks") {
-            utils::g_log << "Deprecated predefinition format '"
-                         << arg << " variable_name=variable_definition'." << endl
-                         << "Use: let(variables_name, variable_definition, expression)" << endl
-                         << "Example: let(h, lmcut(), astar(h))" << endl;
             if (has_search_argument)
                 input_error("predefinitions are forbidden after the '--search' argument");
             if (is_last)
                 input_error("missing argument after " + arg);
             ++i;
             vector<string> predefinition = utils::split(args[i], "=", 1);
-            if (predefinition.size() != 2)
```

Comment:
Why don't we need this check anymore?

Reply:
You are right.
Function: `utils::split(string,separating char, max_nb_splits)`
if we use the split function wit h`max_nb_splits=1`, then it cannot return more than 2 elements, but if the predefinition has no `=`, we would not catch it anymore.